### DISCUSSION TITLE
Not able to run photon error with elastic search

### DISCUSSION THREAD

**Discussion by syedgufran95** [komoot/photon#679] [Q&A]
I have downloaded the data all of it and I am running command 
    java -jar photon-*.jar
in the same directory I have photon_data still I get the error

Error Message

![Screenshot from 2022-07-19 12-50-42](https://user-images.githubusercontent.com/43085010/179690939-6f74b52c-54d5-4bbb-9ea5-df7ef14fd0dd.png)

Command to run photon
![Screenshot from 2022-07-19 12-47-19](https://user-images.githubusercontent.com/43085010/179690442-5defcbbb-db56-4b77-b675-d7e577434854.png)

file path 
![Screenshot from 2022-07-19 12-47-27](https://user-images.githubusercontent.com/43085010/179690499-08338bba-ba61-4aee-887c-5d931f167907.png)

**Comment by lonvia** [comment]
You should never run photon as root. Do you get the same error when running it as a normal user?

**Reply by syedgufran95** [reply]
I tried that as well and it is not working still. Getting the same error

**Reply by lonvia** [reply]
Then you will have to tell us more about the system where you are running python. OS version, JVM version etc.

**Reply by syedgufran95** [reply]
@lonvia 
here you go.
![Screenshot from 2022-07-19 14-06-10](https://user-images.githubusercontent.com/43085010/179706734-c15974b1-82fa-4655-a1c1-6d19d60b69a1.png)

**Reply by lonvia** [reply]
Your system is too old. Please try with a recent Ubuntu, at least 20.04, better 22.04.

PS: please don't post screenshots. They are hard to read and impossible to copy from.
